github-release/orb.yml

```diff
@@ -391,19 +391,18 @@ examples:
         - github/release:
             requires: [Build]
             attach-workspace: true
+            source-env-file: ./build/project.env
             filters:
               branches:
                 only: master
 
-        - before-release:
-            - run:
-                name: Prepare environment for release
-                command: |
-                  cat ./build/project.env >> $BASH_ENV
-                  source $BASH_ENV
-                  tar zcvf ./tmp/artefacts/${BUILD_PROJECT_NAME}_${BUILD_ARCH}_${BUILD_OS_NAME}.tar.gz ./build
+            before-release:
```

Comment:
Does it matter that the indentation has changed here and below?

Reply:
Yes! It got fixed! :)